lib/collection/src/operations/types.rs

```diff
@@ -596,8 +596,8 @@ pub struct DiscoverRequest {
     /// Look for vectors closest to this
     pub target: Option<RecommendExample>,
 
-    /// Pairs of (positive, negative) examples to provide context to the search
-    pub context_pairs: Option<Vec<(RecommendExample, RecommendExample)>>,
+    /// Pairs of [positive, negative] examples to provide context to the search
+    pub context_pairs: Option<Vec<[RecommendExample; 2]>>,
```

Comment:
Let's rename it into just `context` and replace unnamed tuple with explicit data structure.
When I tried to build demo example, I was a bit confused by what expected to go first. 
Also, keeping explicit data structure feels like a safer option for possible further extension.